src/styles/month.scss

```diff
@@ -0,0 +1,11 @@
+@each $n in $month-lenght {
+  .calendar--month-length_#{$n} {
+    @for $i from 1 through $n {
+      >div:nth-child(#{$i}) {
+        &::before {
+          content: "#{$i}";
+        }
+      }
+    }
+  }
+}
```

Comment:
Мне кажется, что здесь можно написать чуть короче и ограничиться одним @each.

Reply:
look in resulting code, now you have like 120 different classes because you have ~30 for each month's length value. i suggest you to add `before` with content for all elements just with plain `for` loop, and then create loop for setting display `none` for deays that are beyond `month-length` value